### PR TITLE
Update readme from experience following it

### PR DESCRIPTION
**Pull request by MichaelRFairhurst**

### CODE REVIEW

File: Dart/README.md [hunk 1]
```diff
@@ -29,16 +29,15 @@ Prerequisites:
...
+   repos have already been cloned you can just update both of them using 'git pull'.
 
 2. Start IntelliJ IDEA Ultimate, on Welcome screen click Configure | Project Defaults | Project Structure | SDKs,
    click [+] and add JDK 1.8. Add [JDK]/lib/tools.jar to the SDK Classpath if it is not there. Rename SDK to 'IDEA jdk'.
    Click [+] to add one more JDK and provide path to JDK 1.8 once again . Leave default name "1.8". Make sure it contains [JDK]/lib/tools.jar
 
 3. On Welcome screen click Configure | Settings (Preferences), look for Path Variables and add the following vars there:
-   - IDEA_ULTIMATE_PLUGINS pointing to [IntelliJ IDEA Ultimate Installation]/Contents/plugins
-     (on Windows: [IntelliJ IDEA Ultimate Installation]/plugins)
+   - IDEA_ULTIMATE_PLUGINS pointing to [IntelliJ IDEA Ultimate Installation]/plugins
```
Comment (alexander-doroshko) [inline Dart/README.md]: Did you setup it on Linux? I believe on Mac the correct path is `[IntelliJ IDEA Ultimate Installation]/Contents/plugins`, but on Windows/Linux: `[IntelliJ IDEA Ultimate Installation]/plugins`

Comment (MichaelRFairhurst) [inline Dart/README.md]: Ah, that also makes sense. I didn't try it on a mac, thought the Content folder was outdated. Will change

**Comment by MichaelRFairhurst** [comment]
Hey, hadn't yet updated it to include 'Content' for Mac -- will open another PR to fix

**Comment by MichaelRFairhurst** [comment]
Ah, looks like someone already handled that. Thanks!

**Pull request merged**